haas/ext/switches/_console.py

```diff
@@ -119,6 +119,13 @@ def revert_port(self, port):
         self.exit_if_prompt()
         self.console.expect(self.config_prompt)
 
+    def _should_save(self, switch_type):
+        """checks the config file to see if switch should save or not"""
+
+        switch_ext = 'haas.ext.switches.' + switch_type
+        return (not cfg.has_option(switch_ext, 'save') or
```

Comment:
If I used `if` I would have to do nested ifs which is [not as elegant](https://github.com/CCI-MOC/hil/pull/749#discussion_r108596058)